lib/i18n/locales/en.ts

```
@@ -0,0 +1,130 @@
+    amount: 'Amount',
+    insufficientBalance: 'Insufficient balance.',
+    available: 'Available',
+    note: 'Note (Optional)',
+    addMessage: 'Add a message...',
+    transferLabel: 'Transfer',
+    dialogDescription: 'Transfer ACBU securely to another wallet',
+    networkFee: 'Network Fee',
+    free: 'Free',
+    cancel: 'Cancel',
+    continue: 'Continue',
+    to: 'To',
```

Biome (2.4.15)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)